jdaviz/core/template_mixin.py

```diff
@@ -1,26 +1,34 @@
-import ipyvuetify as v
+from warnings import warn
+
+from ipyvuetify import VuetifyTemplate
 from glue.core import HubListener
-from traitlets import List
 
 __all__ = ['TemplateMixin']
 
 
-class TemplateMixin(v.VuetifyTemplate, HubListener):
+class TemplateMixin(VuetifyTemplate, HubListener):
     def __new__(cls, *args, **kwargs):
         """
         Overload object creation so that we can inject a reference to the
         ``Hub`` class before components can be initialized. This makes it so
         hub references on plugins can be passed along to components in the
         call to the initialization method.
         """
-        app = kwargs.pop('app', None)
         obj = super().__new__(cls, *args, **kwargs)
-        setattr(obj, '_app', app)
+        obj._app = kwargs.pop('app', None)
 
         return obj
 
     @property
     def app(self):
+        warn(DeprecationWarning('The usage `.app` from a jdaviz application is'
+                                ' deprecated and should be replaced by '
+                                '`.glueapp` to prevent confusion with other '
+                                'meanings of "app"'))
+        return self._app
+
+    @property
+    def glueapp(self):
```

Comment:
Having thought about this a bit more, I misspoke. This would not be access to the `SpecViz` or `CubeViz` helper class instance, but instead the parent jdaviz `Application`, since the plugin instantiate happens there and the helper classes are not subclasses.

```suggestion
        """
        Allows access to the underlying jdaviz application instance. This is **not** access to the helper 
        class, but instead the core application.
        """
```

Reply:
Oops!  This message crossed with the one I just posted - yes, this makes sense now, thanks for clearing that up!